save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt

```diff
@@ -43,31 +44,31 @@ class VulnerabilityService(
 ) {
     /**
      * @param vulnerabilityIdentifier identifier of vulnerability
-     * @param isActive admin vulnerability check flag
+     * @param status status of vulnerability
      * @return vulnerability by name
      */
-    fun findByVulnerabilityIdentifierAndIsActive(vulnerabilityIdentifier: String, isActive: Boolean) = vulnerabilityRepository.findByVulnerabilityIdentifierAndIsActive(
+    fun findByVulnerabilityIdentifierAndStatus(vulnerabilityIdentifier: String, status: VulnerabilityStatus) = vulnerabilityRepository.findByVulnerabilityIdentifierAndStatus(
```

Comment:
Suggestion to implement `VulnerabilityFilter` in order to escape many different methods with long names

Reply:
This is already a big enough refactoring task. Let's leave it for another pr.

Reply:
Ok but create an issue

Reply:
Created